app/models/miq_request.rb

```diff
@@ -480,6 +481,10 @@ def event_name(mode)
     "#{self.class.name.underscore}_#{mode}"
   end
 
+  def process_immediately_on_create?
```

Comment:
I know I came up with this name initially, but looking at it again I think it should just be `process_on_create?`